WordPress/src/main/java/org/wordpress/android/ui/mysite/cards/dashboard/pages/PagesCardBuilder.kt

```diff
@@ -33,9 +34,37 @@ class PagesCardBuilder @Inject constructor(
         )
     }
 
-    @Suppress("UNUSED_PARAMETER")
     private fun getPagesContentItems(pages: List<PagesCardModel.PageCardModel>): List<PageContentItem> {
-        return emptyList()
+        return pages.map{ page ->
+            PageContentItem(
+                title = getPageTitle(page.title),
```

Comment:
FYI and not a np -- I think this may be one of the "context" aware areas that Ovi is talking about; dropping the named parameter. For example dropping "title" when we are using "getPagetTitle" to set it.

Reply:
As discussed, I am not making any changes here in this PR. If, at a later time, the team as a whole decides to make it a standard practice. We will re-visit this 👍🏼 😄